notebooks/official/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -267,7 +267,7 @@
       },
```

Comment:
<p>Not sure if this is possible here, but in this view, all of the links open in the notebook's tab. This might be distracting to customers who are making their way through the notebook then are taken of of their view and out of Git. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Thanks! I've manually changed all the links to make sure they open in a new link (using target="_blank" in HTML)</p>